app/services/resource-actions.js

```diff
@@ -25,6 +25,7 @@ export default Ember.Service.extend({
       $toggle.removeClass('open');
       $menu.addClass('hide');
       this.set('open', false);
+      this.set('model', null);
```

Comment:
Doesn't look like it's your fault but while you're here, the indent level is wrong in this block.